openy.post_update.php

```diff
@@ -6,19 +6,23 @@
  */
 
 /**
-* Cleanup upgrade path message errors that we cannot prevent from being created.
-*/
+ * Cleanup upgrade path message errors.
+ *
+ * We cannot prevent this upgrade path messages from being created.
+ */
 function openy_post_update_cleanup_update_path_messages() {
-  // Here we clean up messages that appeared in the message log during last 5 minutes.
-  // These messages are generated by node_post_update_configure_status_field_widget() and cannot be handled by upgrade path tool,
-  // because Drupal Core automatically increases weght which doesn't match database.
+  // Here we clean up messages that appeared in the message log during
+  // last 5 minutes. These messages are generated by
+  // node_post_update_configure_status_field_widget() and cannot be handled
+  // by upgrade path tool, because Drupal Core automatically
+  // increases weight which doesn't match database.
   // @see node_post_update_configure_status_field_widget()
   $range = strtotime("-5 minutes");
-  $query = \Drupal::entityQuery('logger_entity')
+  $query = \Drupal::entityQuery('openy_upgrade_log')
     ->condition('name', 'core.entity_form_display.node', 'STARTS_WITH')
     ->condition('created', $range, '>');
   $ids = $query->execute();
-  $storage_handler = \Drupal::entityTypeManager()->getStorage('logger_entity');
```

Comment:
keep it please

Reply:
`logger_entity` not used in Open Y code anymore

Reply:
It is used in OpenY instances, can't be deprecated or removed @hamrant